solver/pb/ops.proto

```diff
@@ -358,6 +360,13 @@ message FileActionMkFile {
 	int64 timestamp = 5;
 }
 
+message FileActionSymlink {
+	// destination path for the new file representing the link
+	string target = 1;
+	// source path for the link
+	string linkPath = 2;
+}
```

Comment:
WDYT about updating this to allow for also creating hardlinks? Similar to `ln` vs `ln -s`? Or at least leaving that door open by naming this `FileActionLink`?

Reply:
I wouldn't put symlinks and hardlinks in the same PR.